bazel/docker/README.md

```diff
@@ -0,0 +1,11 @@
+# How to use this Dockerfile (Dockerfile.clang-8.ubuntu18.04)
+1. Create a dir where you want to build eg: docker_demo/build
+2. Download ngraph-bridge repo : git clone https://github.com/tensorflow/ngraph-bridge.git
+3. docker build -t <ngtf_build> -f Dockerfile.clang-8.ubuntu18.04 .
+4. docker run -it -v ${PWD}:/<workspace> -w /<workspace> <ngtf_build>
+5. Create a virtual env : virtualenv -p python3 <venv3>
+6. Activate the virtual env : source venv3/bin/activate
+7. pip install future
```

Comment:
Just making sure. There are 2 python packages, `future` and `futures`. We want `future` here, right?

Reply:
Yes